lib/ansible/modules/windows/win_find.ps1

```diff
@@ -211,9 +237,13 @@ Function Assert-FileStat($info) {
     $hidden_match = Assert-Hidden -info $info
     $pattern_match = Assert-Pattern -info $info
     $size_match = Assert-Size -info $info
+    
 
     if ($age_match -and $file_type_match -and $hidden_match -and $pattern_match -and $size_match) {
```

Comment:
That implementation requires that other parameters (not only `contains` will be used/fulfilled). IMHO it's not fully correct because it should be possible to use the `contains` parameter separately.